WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardViewModel.swift

```diff
@@ -290,6 +316,13 @@ private extension BlazeCampaignDashboardViewModel {
             }
             .store(in: &subscriptions)
     }
+    func trackingSyncingResult() {
```

Comment:
nit: Would renaming into `trackSyncingResult` make sense? 

nit: New line needed above.

Reply:
Updated in 3c2a878004db0504ceea99840c8cf3af259b8705 👍